fontc/src/lib.rs

```diff
@@ -3117,4 +3118,59 @@ mod tests {
             .iter()
             .all(|instance| instance.unwrap().post_script_name_id.is_none()));
     }
+
+    #[rstest]
+    #[case::yes(true)]
+    #[case::no(false)]
+    fn rename_glyphs_to_production_names(#[case] use_production_names: bool) {
+        let result = TestCompile::compile("glyphs3/ProductionNames.glyphs", |mut args| {
+            args.no_production_names = !use_production_names;
```

Comment:
Neither use_production_names nor do_production_names tell me much about what actually happens when reading. Might we name more informatively? `adobe_style_post_names` or something?